docs/src/upgrade/release-3.7.x.asciidoc

```diff
@@ -206,11 +206,12 @@ String index out of range: -2
 Type ':help' or ':h' for help.
 ----
 
-The `substring()`-step will begin at the start index and return a substring with the length specified. Negative start
-indices are allowed and will begin at the specified index counted from the end of the string:
+The `substring()`-step will return a substring with indices specified by the start and end indices, or from
+the start index to the remainder of the string if an end index is not specified. Negative indices are allowed and will
+begin at the specified index counted from the end of the string:
```

Comment:
```suggestion
the start index to the remainder of the string if an end index is not specified. Negative indices are allowed and will
count from the end of the string:
```
Suggest removing "begin" as that specifically refers to the start index

Reply:
Committed suggestion